src/containers/main-page/stepper/index.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { FC } from 'react';
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import { Box, Typography } from '@mui/material';
+import { StepConnectorStyled, StepLabelStyled } from './styled';
+import StepperControls from './StepperControls';
+import { CV_FORM_STEPS, getFormByStep } from './utils/config';
+import { useForm } from '../../../common/utils/hooks';
+
+const CustomStepper = function (): JSX.Element {
+  const { value, handleChange } = useForm();
+  const [activeStep, setActiveStep] = React.useState(0);
+
+  const stepsArrayRaw = Object.values(CV_FORM_STEPS);
+  const steps = stepsArrayRaw.slice(0, (stepsArrayRaw.length / 2));
```

Comment:
What is ` 2 ` here?

Reply:
Because Object.values from CV_FORM_STEPS provides entire map-like enum structure with output like: 
['Step1', 'Step2, 'Step3', 0 , 1 , 2]